src/sentry/static/sentry/app/components/sidebar/index.jsx

```diff
@@ -548,6 +549,19 @@ class Sidebar extends React.Component {
 
         {hasOrganization && (
           <SidebarSectionGroup>
+            {window.localStorage.getItem('NEW_ONBOARDING_SIDEBAR') && !horizontal && (
```

Comment:
We could use https://github.com/getsentry/sentry/blob/d7b6c6ef700e4e01f23bbe71e2eca9ae1c5aaae8/src/sentry/static/sentry/app/utils/localStorage.tsx#L1-L38

Reply:
Nice, this was already imported.